Controller/Adminhtml/Configuration/MerchantAccounts.php

```diff
@@ -71,7 +72,7 @@ public function execute()
 
             $resultJson->setData($response);
             return $resultJson;
-        } catch (AdyenException $e) {
+        } catch (AdyenException | ConnectionException $e) {
```

Comment:
`ConnectionException` is thrown when something is wrong on the connection to Adyen so in this case it should not return 4XX response status code as it's not a client error, it should return a 5XX in this case

It would be more convenient to catch it in another catch statment

```php
     } catch (ConnectionException $e) {
            $resultJson->setHttpResponseCode(400);
            ...
```